docs/walk-through/parameters.md

```diff
@@ -89,3 +89,34 @@ spec:
 ```
 
 In this workflow, both steps `A` and `B` would have the same log-level set to `INFO` and can easily be changed between workflow submissions using the `-p` flag.
+
```

Comment:
```suggestion

# `enum` support

> v3.0 and after

```

Let's put this in a new section since it is distinct from the rest.

We can also add a version specifier for consistency with the rest of the docs. #4314 was released as part of [v2.12](https://github.com/argoproj/argo-workflows/blob/v3.5.4/CHANGELOG.md#v2120-rc1-2020-11-06), while #4365 was released as part of [v3.0](https://github.com/argoproj/argo-workflows/blob/v3.5.4/CHANGELOG.md#v300-rc1-2021-02-08)